### ISSUE TITLE
API Specification

### ISSUE THREAD

**Issue by weiss-d** [NamkhaEncyclopedia/calculator-webapp-backend#2]
# API functions (PR: https://github.com/NamkhaEncyclopedia/calculator-webapp-backend/pull/6)
1) For given date, time and geo coordinates get an offset from UTC in seconds for that geographic point at that time.

2) For given:
- `calculation_method`: `rinpoche` | `classic`
- `namkha_type`: `year` | `month` | `day` | `hour`
- `name` (optional)
- `gender`: `female` | `male`
- date, time, UTC offset
- geo coordinates

provide:
- pictures of calculation result (one picture per page, 2 pages, format: PNG or WebP)
- download link for a PDF file with calculation result.

**Comment by stasdavydov** [comment]
As soon as files generation take time we need to make calculation asynchronous:
1. At first we request to start generation by parameters -> and receive request ID.
2. Then we request generated files by this ID when they are ready.

**Comment by stasdavydov** [comment]
One more question about `gender` -- does it affect the calculation? What the values for the `gender`?
Predict we may get angry feedback if we have `M` and `F` only  :see_no_evil:

**Comment by stasdavydov** [comment]
@weiss-d  I remember it's possible to calculate namkha not for a human, but for a family or organization, do I remember right? What we shall use for `gender` in this case? Is it optional?

**Comment by stasdavydov** [comment]
@weiss-d Why should we pass LAT and LON? We already have date and time and TZ obtained separately from another endpoint. Right?

**Comment by weiss-d** [comment]
@stasdavydov 

> One more question about `gender` -- does it affect the calculation? What the values for the `gender`? Predict we may get angry feedback if we have `M` and `F` only 🙈

Yes, it affects position of Fortune and Capacity aspect (left or right) and corresponding Mewa aspects. I'll specify it in render module.



> [@weiss-d](https://github.com/weiss-d) I remember it's possible to calculate namkha not for a human, but for a family or organization, do I remember right? What we shall use for `gender` in this case? Is it optional?

In this case a person requesting the calculation should decide on 'gender'. Anna Weiss will elaborate on this in Help section of the app page.



> [@weiss-d](https://github.com/weiss-d) Why should we pass LAT and LON? We already have date and time and TZ obtained separately from another endpoint. Right?

We need it to calculate twilight time, I'll put more 
info about it in Notebooks in the future.
